core/commands/snapshot_command.rb

```diff
@@ -309,11 +310,14 @@ def revert_snapshot(node_name, snapshot_name)
     snapshots = get_snapshots(node_name)
     raise SNAPSHOTS_NOT_FOUND if snapshots.empty?
     raise SNAPSHOT_NOT_EXISTS unless snapshots.include? full_snapshot_name
+
     case @provider
     when LIBVIRT
-      run_reliable_command("virsh snapshot-revert --domain #{@nodes_directory_name}_#{node_name} --snapshotname #{full_snapshot_name}")
       pwd = Dir.pwd
       Dir.chdir @path_to_nodes
```

Comment:
The `node_running?` method clearly depends on the configuration directory. Either make the Vagrant method aware of the need for the working directory change or notify users of this method to change directory. I prefer the first variant.